components/ipfs/ipfs_utils.cc

```diff
@@ -5,36 +5,25 @@
 
 #include "brave/components/ipfs/ipfs_utils.h"
 
-#include <optional>
 #include <string>
 #include <string_view>
 #include <vector>
 
-#include "base/feature_list.h"
+#include "base/logging.h"
 #include "base/strings/string_split.h"
 #include "base/strings/string_util.h"
 #include "base/strings/stringprintf.h"
 #include "brave/components/filecoin/rs/src/lib.rs.h"
-#include "brave/components/ipfs/features.h"
-#include "brave/components/ipfs/ipfs_constants.h"
-#include "brave/components/ipfs/ipfs_ports.h"
-#include "brave/components/ipfs/keys/ipns_keys_manager.h"
-#include "brave/components/ipfs/pref_names.h"
 #include "components/base32/base32.h"
-#include "components/prefs/pref_service.h"
 #include "net/base/registry_controlled_domains/registry_controlled_domain.h"
 #include "net/base/url_util.h"
-#include "third_party/re2/src/re2/re2.h"
 #include "url/gurl.h"
 #include "url/origin.h"
 
 namespace {
-
-// RegEx to validate the node name:
-// go-ipfs_v0.9.0-rc1_windows-amd64 - valid
-// go-ipfs_v0.9.0_windows-amd64 - valid
-constexpr char kExecutableRegEx[] =
-    "go-ipfs_v(\\d+\\.\\d+\\.\\d+)(-rc\\d+)?\\_\\w+-\\w+";
+constexpr char kIPFSScheme[] = "ipfs";
+constexpr char kIPNSScheme[] = "ipns";
+constexpr char kDefaultPublicGateway[] = "https://ipfs.io";
```

Comment:
this url will be used both on the NFT page and for the ENS resolver. Should it be same? Shouldn't it be configurable?